MDM/Jamf/zz_Quit_SwiftDialog.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/zsh
+
+# MARK: Arguments/Parameters
+
+# Parameter 4: path to the swiftDialog command file
+dialog_command_file=${4:-"/var/tmp/dialog.log"}
+
+# MARK: Constants
+dialogApp="/Library/Application Support/Dialog/Dialog.app"
+
+dialogUpdate() {
+    # $1: dialog command
+    local dcommand="$1"
+
+    if [[ -n $dialog_command_file ]]; then
+        echo "$dcommand" >> "$dialog_command_file"
+        echo "Dialog: $dcommand"
+    fi
+}
+
+# check minimal macOS requirement
+if [[ $(sw_vers -buildVersion ) < "20A" ]]; then
+    echo "This script requires at least macOS 11 Big Sur."
+    exit 98
+fi
+
+# check we are running as root
+if [[ $DEBUG -eq 0 && $(id -u) -ne 0 ]]; then
+    echo "This script should be run as root"
+    exit 97
+fi
+
+# check for Swift Dialog
+if [[ ! -d $dialogApp ]]; then
+    echo "Cannot find dialog at $dialogApp"
+    exit 95
+fi
+
+
+# close and quit dialog
+dialogUpdate "progress: complete"
+dialogUpdate "progresstext: Done"
+
+# pause a moment
+sleep 0.5
+
+dialogUpdate "quit:"
+
+# just to be safe
+killall "Dialog"
+
```

Comment:
I think the killall is running before dialog gets a chance to quit by itself. This will leave an orphaned `tail -f` process but I might re-check that as I had to change some of the logic around that for the issue Brad reported and the `quit:` command isn't cleaning up. if you've done a bit of testing you probably have a few instances of `tail` running

Reply:
I actually do see the tail processes. So remove the `killall`?

Reply:
inserted a `sleep 0.5` https://github.com/Installomator/Installomator/pull/641/commits/ef19aa28fe48528b9677992bccca10d145d9afe6